.github/workflows/pull_request.yaml

```diff
@@ -19,17 +19,17 @@ jobs:
     - uses: actions/checkout@v3
       with:
         fetch-depth: 0
-    - name: Setup GCloud Auth
-      uses: google-github-actions/auth@v1
+    - name: Set up Python
+      uses: actions/setup-python@v4
       with:
-        project_id: ${{ secrets.GCP_PROJECT_ID }}
-        credentials_json: ${{ secrets.GCP_SA_KEY }}
-        create_credentials_file: true
-        export_environment_variables: true
-    - name: Setup GCloud SDK
-      uses: google-github-actions/setup-gcloud@v1.1.0
+        python-version: 3.8
+    - uses: google-github-actions/setup-gcloud@a48b55b3b0eeaf77b6e1384aab737fbefe2085ac
       with:
-        version: '>= 386.0.0'
+        version: '290.0.1'
```

Comment:
why use an older version?

Reply:
ah i see https://github.com/solo-io/go-utils/pull/512/files#r1183133800

Reply:
See my [comment above](https://github.com/solo-io/go-utils/pull/512#discussion_r1183133800). The current action is failing unexpectedly with an error I have not seen before and which I could not solve quickly. I could not afford to spend more time to look into this, so I just used the same setup that we use in other repos currently and that I know works. Long term we should switch to using identity federation instead of relying on these API keys.